Tasks/AzureRmWebAppDeployment/task.loc.json

```diff
@@ -168,13 +205,8 @@
   ],
   "instanceNameFormat": "ms-resource:loc.instanceNameFormat",
   "execution": {
-    "AzurePowerShell": {
```

Comment:
Since we are removing handler from task.json, we can remove corresponding .ps1 files too.